modules/skybrowser/src/screenspaceskybrowser.cpp

```diff
@@ -50,25 +50,25 @@ namespace {
 
     constexpr const openspace::properties::Property::PropertyInfo RenderCopyInfo = {
         "RenderCopy",
-        "RAE Position Of A Copy Of The Sky Browser",
+        "Position Of A Copy Of The Sky Browser",
```

Comment:
Wondering if this should be "Position of a Copy of the Sky Browser", or even better, maybe we should come up with a shorter name. "Sky Browser Copy Position"? or just "Render Copy Position", as the description says what it is 

We should probably also decide on whether it is a "RenderCopy" or "DisplayCopy" (the Gui says Display Copy
)